src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandlerTest.java

```diff
@@ -36,14 +36,14 @@
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "indentation" + File.separator + filename);
+                + "indentation" + File.separator + "newhandler" + File.separator + filename);
     }
 
     @Test
     public void testInvalidLabel() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(IndentationCheck.class);
```

Comment:
lets do this in scope of this issue. subkrish is not a newbie in our project he could make it. Fix is simple.